x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/persistence/JobResultsProviderTests.java

```diff
@@ -1031,6 +855,31 @@ public void testDatafeedTimingStats_NotFound() throws IOException {
         verifyNoMoreInteractions(client);
     }
 
+    @SuppressWarnings("unchecked")
+//    public void testTermFieldMapping() throws IOException {
```

Comment:
I just noticed this one, I forgot to address it. I'll take care of it before removing the WIP label.